src/Cli/dotnet/commands/dotnet-sln/list/SlnListParser.cs

```diff
@@ -11,6 +12,8 @@ namespace Microsoft.DotNet.Cli
 {
     public static class SlnListParser
     {
+        public static readonly Option<bool> SolutionFolderOption = new Option<bool>(new string[] { "-s", "--solution-folders" }, LocalizableStrings.ListSolutionFoldersArgumentDescription);
```

Comment:
I kind of prefer `-f` because it enunciates the 'folder' aspect of it, but this is also good.

Reply:
@baronfel to comment on the choice of short parameter.